### DISCUSSION TITLE
Z offset

### DISCUSSION THREAD

**Discussion by anoblet** [mriscoc/Ender3V2S1#1060] [Q&A]
Is there a possibility to set the z offset in the thousandths?

**Comment by Micah-Scheck** [comment]
I don't really have too much experience, but I don't know if that would work. The stepper motors already do micro steps for anything under .04mm so I don't know if the printer would even have the accuracy to extend to thousandths.

**Reply by anoblet** [reply]
It already rounds up, and I've used `M851 Z-2.425. `
